ldap_fluff.gemspec

```
@@ -4,7 +4,7 @@ Gem::Specification.new do |s|
   s.summary     = 'LDAP querying tools for Active Directory, FreeIPA and POSIX-style'
   s.description = 'Simple library for binding & group querying on top of various LDAP implementations'
   s.homepage    = 'https://github.com/theforeman/ldap_fluff'
-  s.license     = 'GPLv2'
+  s.license     = 'GPL-3.0'
```

Changing the license is a major thing. You can't just do that.

The warning is about the format. Fedora has had its own license codes that predates [SPDX](https://spdx.dev/) formats. Fedora is also standardizing on SPDX and https://docs.fedoraproject.org/en-US/legal/allowed-licenses/ provides a translation.
```suggestion
  s.license     = 'GPL-2.0-only'
```

---

As mentioned during the call, https://spdx.org/licenses/ is the full overview. The current license is https://spdx.org/licenses/GPL-2.0-only.html but the PR is changing it to https://spdx.org/licenses/GPL-3.0-only.html which is legally not allowed with explicit approval of all previous contributors. The easiest way to solve it now to use the proper SPDX tag.

---

Thanks, I'll keep this in mind.